src/shared/compatibility.js

```diff
@@ -24,7 +24,8 @@ if ((typeof PDFJSDev === 'undefined' ||
 var globalScope =
   (typeof window !== 'undefined' && window.Math === Math) ? window :
   (typeof global !== 'undefined' && global.Math === Math) ? global :
-  (typeof self !== 'undefined' && self.Math === Math) ? self : this;
+  (typeof self !== 'undefined' && self.Math === Math) ? self :
+  (typeof this !== 'undefined' && this.Math === Math) ? {};
```

Comment:
This fails linting, see https://travis-ci.org/mozilla/pdf.js/builds/262068404, since this line seems to contain a SyntaxError. I'm assuming that what you actually want here is `(typeof this !== 'undefined' && this.Math === Math) ? this : {};` instead :-)

Please note that you can easily run linting locally (using `gulp lint`) to catch these kind of issues, and please remember to squash the commit when fixing this.